website/templates/util/render_node.mako

```diff
@@ -51,16 +51,30 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
             <div class="pull-right">
                 % if not summary['primary'] and 'write' in user['permissions'] and not node['is_registration']:
                     <i class="fa fa-times remove-pointer" data-id="${summary['id']}" data-toggle="tooltip" title="Remove link"></i>
                     <i class="fa fa-code-fork" onclick="NodeActions.forkPointer('${summary['id']}', '${summary['primary_id']}');" data-toggle="tooltip" title="Fork this ${summary['node_type']} into ${node['node_type']} ${node['title']}"></i>
                 % endif
-                <i id="icon-${summary['id']}" class="pointer fa fa-angle-down" onclick="NodeActions.openCloseNode('${summary['id']}');" style="font-weight:bold;"></i>
+                % if summary['node_type'] == 'component' and summary['logged_in'] and summary['is_contributor']:
```

Comment:
Speaking of Registrations (re archiving), you'll need to check with product whether they want these buttons to show up at all for registrations. At a minimum, the delete button should be removed as you cannot delete registrations and it fails when you try.